compiler-core/src/language_server/completer.rs

```diff
@@ -16,7 +16,8 @@ use crate::{
     line_numbers::LineNumbers,
     type_::{
         self, collapse_links, pretty::Printer, AccessorsMap, FieldMap, ModuleInterface,
-        PreludeType, Type, TypeConstructor, ValueConstructorVariant, PRELUDE_MODULE_NAME,
+        PreludeType, Type, TypeAliasConstructor, TypeConstructor, ValueConstructorVariant,
```

Comment:
How come there are changes in this file? I wouldn't have expected there to be changes to language server code to enable exporting the package interface from cache

Reply:
It's because, as types and type aliases are now stored separately, we need to account for them separately in the completion

Reply:
Thank you